tests/render/test_inline_renderer.py

```diff
@@ -0,0 +1,361 @@
+from typing import List
+
+import pytest
+
+from great_expectations.core.batch import BatchRequest
+from great_expectations.core.expectation_configuration import ExpectationConfiguration
+from great_expectations.core.expectation_suite import ExpectationSuite
+from great_expectations.core.expectation_validation_result import (
+    ExpectationValidationResult,
+)
+from great_expectations.data_context import DataContext
+from great_expectations.render.renderer.inline_renderer import InlineRenderer
+from great_expectations.render.types import RenderedAtomicContent
+from great_expectations.validator.validator import Validator
+
+
+@pytest.mark.parametrize(
+    "expectation_configuration,expected_serialized_expectation_configuration_rendered_atomic_content,expected_serialized_expectation_validation_result_rendered_atomic_content",
+    [
+        pytest.param(
+            ExpectationConfiguration(
+                expectation_type="expect_table_row_count_to_equal",
+                kwargs={"value": 3},
+            ),
```

Comment:
Could we give each of these params an id or name so we're able to access them with `pytest -k`?

Reply:
I was about to write the same thing 🙇

Reply:
I added an id to each of the test cases and confirmed it works with `pytest -k` ✅